### DISCUSSION TITLE
undefined reference to 'strtof' error from protobuf lib when using minSdkVersion 19

### DISCUSSION THREAD

**Discussion by naahmedea** [android/ndk#1784] [Q&A]
Description
When I use minSdkVersion 19 compile my project, and in the link step, I got this error:
protobuf/android/armeabi-v7a/libProtobuf.a(strutil.cc.o):strutil.cc:function protobuf::safe_strtof(char const*, float*): error: undefined reference to 'strtof'
But I change my minSdkVersion to 21, the error disappear.
My APP must support API 19, Do we have some workaround to solve this problem?

Environment Details
NDK Version: 19, 21 ( I have tried some version, got the same error)
Build system: cmake
ABI: armeabi-v7a
STL: c++static

**Comment by DanAlbert** [comment]
There isn't really any information here. File a bug with a repro case.
